packages/plugins/RedPacket/src/SiteAdaptor/utils/getRedPacketCover.ts

```diff
@@ -0,0 +1,19 @@
+import urlcat from 'urlcat'
+
+interface Options {
+    theme: 'mask' | 'golden-flower' | 'lucky-flower' | 'lucky-firefly' | 'co-branding'
+    symbol: string
+    shares: number | string
+    amount: number | string
+    from: string
+    message: string
+    remainingAmount: number | string
+    remainingShares: number | string
+}
+
+export function getRedPacketCover(options: Options) {
```

Comment:
We can handle the fetching of cover in the FireflyRedpacket api

Reply:
This is not fetching but constructing the url